<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CPfmTable</class>
 <widget class="QWidget" name="CPfmTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_cpu">
       <property name="text">
        <string>CPU</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_update_time">
       <property name="text">
        <string>update time : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_time">
       <property name="text">
        <string>00:00:00.00</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <row>
      <property name="text">
       <string>New Row</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>New Row</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>New Row</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>New Row</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>New Row</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>New Row</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>New Row</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>New Row</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>New Row</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>New Row</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>New Row</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>New Row</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>New Row</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>New Row</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>New Row</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>New Row</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>GROUP</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>CH</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>SV</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>RV</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>RC</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>STATUS</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>PPS</string>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item row="0" column="2">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="0" column="3">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="0" column="4">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="0" column="5">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item row="1" column="2">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="1" column="3">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="1" column="4">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="1" column="5">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="2" column="1">
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item row="2" column="2">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="2" column="3">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="2" column="4">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="2" column="5">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="3" column="1">
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item row="3" column="2">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="3" column="3">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="3" column="4">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="3" column="5">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="4" column="1">
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item row="4" column="2">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="4" column="3">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="4" column="4">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="4" column="5">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="5" column="1">
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item row="5" column="2">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="5" column="3">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="5" column="4">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="5" column="5">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="6" column="1">
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item row="6" column="2">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="6" column="3">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="6" column="4">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="6" column="5">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="7" column="1">
      <property name="text">
       <string>8</string>
      </property>
     </item>
     <item row="7" column="2">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="7" column="3">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="7" column="4">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="7" column="5">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="8" column="0">
      <property name="text">
       <string>S1PS</string>
      </property>
     </item>
     <item row="8" column="1">
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item row="8" column="2">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="8" column="3">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="8" column="4">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="8" column="5">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="9" column="1">
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item row="9" column="2">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="9" column="3">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="9" column="4">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="9" column="5">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="10" column="1">
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item row="10" column="2">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="10" column="3">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="10" column="4">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="10" column="5">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="11" column="1">
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item row="11" column="2">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="11" column="3">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="11" column="4">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="11" column="5">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="12" column="1">
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item row="12" column="2">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="12" column="3">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="12" column="4">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="12" column="5">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="13" column="1">
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item row="13" column="2">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="13" column="3">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="13" column="4">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="13" column="5">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="14" column="1">
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item row="14" column="2">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="14" column="3">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="14" column="4">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="14" column="5">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="15" column="1">
      <property name="text">
       <string>8</string>
      </property>
     </item>
     <item row="15" column="2">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="15" column="3">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="15" column="4">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="15" column="5">
      <property name="text">
       <string>-</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
